Description: Use system yasm instead of building included copy.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2015-03-08

--- a/configure.in
+++ b/configure.in
@@ -936,7 +936,7 @@
     CALLING_CONVENTIONS_OBJS='x86call.lo x86check$U.lo'
 
     # Currently yasm is only needed to build x86 assembly files on some hosts
-    want_yasm="yes"
+    want_yasm="no"
     # Availability of rdtsc is checked at run-time.
     SPEED_CYCLECOUNTER_OBJ=pentium.lo
     
@@ -1677,7 +1677,7 @@
 GMP_CHECK_LIBM_FOR_BUILD
 
 
-MPIR_AS="/yasm/yasm"
+MPIR_AS="yasm"
 
 # How to assemble, used with CFLAGS etc, see mpn/Makeasm.am.
 # Using the compiler is a lot easier than figuring out how to invoke the
@@ -2980,7 +2980,6 @@
 #        Right now automake isn't accepting the new AC_CONFIG_FILES scheme.
 
 #     allways configure all subdirectorys , even if there are not going to be built , 
-AC_CONFIG_SUBDIRS([yasm])
 
 case $host in
   *-w64-mingw*)
--- a/mpn/Makeasm.am
+++ b/mpn/Makeasm.am
@@ -119,4 +119,4 @@
 #  .as assembler, assembled with Yasm
 
 .as.lo:
-	$(LIBTOOL) --mode=compile --tag=CC $(top_srcdir)/strip_fPIC.sh $(top_builddir)$(MPIR_AS) -I $(top_builddir) $(OBJECT_FORMAT) $(GSYM_FLAG) -o $@ `test -f '$<' || echo '$(srcdir)/'`$<
+	$(LIBTOOL) --mode=compile --tag=CC $(top_srcdir)/strip_fPIC.sh $(MPIR_AS) -I $(top_builddir) $(OBJECT_FORMAT) $(GSYM_FLAG) -o $@ `test -f '$<' || echo '$(srcdir)/'`$<
